tyk-headless/templates/deployment-gw-repset.yaml

```
@@ -71,6 +71,8 @@ spec:
         env:
           - name: TYK_GW_LISTENPORT
             value: "{{ .Values.gateway.containerPort }}"
+          - name: TYK_GW_CONTROLAPIPORT
+            value: "{{ .Values.gateway.control.containerPort }}"
```

I think this needs an `if` so that the config is default if the control port is not enabled:
```suggestion
          {{- if .Values.gateway.control.enabled }}
          - name: TYK_GW_CONTROLAPIPORT
            value: "{{ .Values.gateway.control.containerPort }}"
          {{- end }}
```

---

set IFF enabled